docs/DOCUMENTING.md

```diff
@@ -85,9 +85,12 @@ the provided data needs to conform to various CUE schemas. To check the validity
 of the CUE sources:
 
 ```bash
-make check-docs
+cd .. # Change to the repo root directory
+CI=true make check-docs
 ```
 
+> When the CI flag is on, then the checker will also run a CUE format validation step.
```

Comment:
This is normally skipped because it makes modifications to files. One thing we could do is change the check so that it doesn't modify files (maybe by writing to a temp directory) and then we wouldn't need to guard it with `CI=true`.

Reply:
Good point. Added extra context here to prevent surprises.